src/lib/services/paths-service.ts

```
@@ -0,0 +1,41 @@
+}
+
+export function getStacksDirectory(customPath?: string): string {
+	return customPath || path.resolve(BASE_PATH, 'stacks');
+}
+
+// Default stacks directory
+export const DEFAULT_STACKS_DIR = path.resolve(BASE_PATH, 'stacks');
+
+// Helper function to ensure directories exist with proper permissions
+export async function ensureDirectory(dir: string, mode = 0o755): Promise<void> {
+	const fs = await import('fs/promises');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)